src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -625,7 +627,9 @@ private CustomResourceSnapshot ToSnapshot(Container container, CustomResourceSna
                 new(KnownProperties.Container.Ports, GetPorts()),
             ],
             EnvironmentVariables = environment,
-            CreationTimeStamp = container.Metadata.CreationTimestamp?.ToLocalTime(),
+            CreationTimeStamp = container.Metadata.CreationTimestamp?.ToUniversalTime(),
+            StartupTimeStamp = container.Status?.StartupTimestamp?.ToUniversalTime().UtcDateTime,
```

Comment:
@karolz-ms CreationTimestamp is a DateTime and StartupTimestamp is a DateTimeOffset. Do you know why the property types are different? Can they and should they be made to have a consistent type?

Reply:
`CreationTimestamp` comes from `V1ObjectMeta`. Since the built-in types use DateTime, should our custom properties be consistent with them?

Reply:
The date doesn't have an offset, so I'm guessing it comes back as UTC. Deserializing to DateTime should be fine. What do you think?

![image](https://github.com/user-attachments/assets/1d52a581-02d4-49ac-992f-14034cb10da4)

Reply:
I think whatever makes things more consistent is fine. I suggested `DateTimeOffset` initially because the K8s code is using Go time, which always has a location associated with it https://pkg.go.dev/time@go1.23.1#Time. But if C# Kubernetes client uses DateTime for timestamps, we should probably follow that.